core/cmd/app.go

```diff
@@ -907,8 +907,14 @@ func NewApp(client *Client) *cli.App {
 			Subcommands: []cli.Command{
 				{
 					Name:   "create",
-					Usage:  "Send <amount> Eth from node ETH account <fromAddress> to destination <toAddress>.",
+					Usage:  "Send <amount> ETH (actual ETH, not wei) from node ETH account <fromAddress> to destination <toAddress>.",
```

Comment:
Since we already support, and expect, the `amount` argument to be passed,I added support for bool flags to state whether you'd like to send WEI or ETH via the CLI

Reply:
So the command would have `-amount 100000000 -wei`, and shouldn't have to pass `-eth` to actually send ETH since it would be the default behavior.